AFQ/api.py

```diff
@@ -282,17 +289,17 @@ def __init__(self,
         self.use_prealign = (use_prealign and (self.reg_algo != 'slr'))
         self.b0_threshold = b0_threshold
 
-        self.scalars = scalars
+        self.scalars = [scalar.lower() for scalar in scalars]
 
         if virtual_frame_buffer:
             from xvfbwrapper import Xvfb
             self.vdisplay = Xvfb(width=1280, height=1280)
             self.vdisplay.start()
-        self.viz = Viz(backend=viz_backend)
+        self.viz = Viz(backend=viz_backend.lower())
 
         default_tracking_params = get_default_args(aft.track)
-        default_tracking_params["seed_mask"] = ScalarMask("dti_fa")
-        default_tracking_params["stop_mask"] = ScalarMask("dti_fa")
+        default_tracking_params["seed_mask"] = ScalarMask(self.scalars[0])
```

Comment:
Ooh. This can be rather brittle. What if a user asks for `["dti_md", "dti_fa"]` as scalars?

Reply:
Then "dti_md" would be the seed mask. I think its more brittle, but I don't know what the smart default is for this. Maybe look in scalars for a string containing fa, and if its not there, then use the first one? I want this to be more automatic, such that when the user inputs ["dki_fa", "dki_md"] she doesnt have to change a bunch more parameters.